gctscript/modules/ta/indicators/index.go

```diff
@@ -0,0 +1,68 @@
+package indicators
+
+import (
+	"fmt"
+
+	objects "github.com/d5/tengo/v2"
+	"github.com/thrasher-corp/go-talib/indicators"
+	"github.com/thrasher-corp/gocryptotrader/gctscript/modules"
+)
+
+// IndexModule index indicator commands
+var IndexModule = map[string]objects.Object{
+	"mfi": &objects.UserFunction{Name: "mfi", Value: mfi},
+}
+
+func mfi(args ...objects.Object) (objects.Object, error) {
+	if len(args) != 2 {
+		return nil, objects.ErrWrongNumArguments
+	}
+
+	ohlcvInput := objects.ToInterface(args[0])
+	ohlcvInputData := ohlcvInput.([]interface{})
+	ohclvData := make([][]float64, 6)
```

Comment:
why are we padding the slice with 2 extra elements, is this to do with how we add to the objects?

Reply:
I tried to keep it consistent with all usage of input OHLCV data as a 
```
ohclvData[0] = tmestamp 
ohclvData[1] = open
ohclvData[2]  = high
ohclvData[3] = low
ohclvData[4]  = close
ohclvData[5] = vol
```
And when its called you end up with 	

```
ret := indicators.Atr(ohclvData[2], ohclvData[5], ohclvData[4], inTimePeriod)
```

that way you can always assume that ohclvData[2] = high data no matter what indicator

It does mean there are cases like this where i am not passing the data so there will be empty slices (timestamp / open for example)

I am happy to make any changes to his for user experience improvements

Reply:
Ahh okay that makes sense. What's the performance hit if we just use a slice of an indicator OHLC object with all those datapoints?

Reply:
Btw I am happy with keeping as is, its not cruicial, just a bit easier to understand if other people want to add in indicators.

Reply:
as in update the indicator on the go-talib side to accept the entire OHLCV slice and handle it there instead? 

That could work out better for users

Reply:
I was thinking of something else but that works also! 👍